### ISSUE TITLE
Can't tell if it is training?

### ISSUE THREAD

**Issue by matt3209** [junyanz/pytorch-CycleGAN-and-pix2pix#1470]
<img width="1429" alt="Screen Shot 2022-08-15 at 10 53 26 PM" src="https://user-images.githubusercontent.com/52147717/184794476-96432ac6-f2fa-4b2a-bdca-bb61deeef9cc.png">
<img width="344" alt="Screen Shot 2022-08-15 at 10 53 15 PM" src="https://user-images.githubusercontent.com/52147717/184794482-5cabe5cf-4bbb-4f81-9efb-d6f185c82654.png">
<img width="922" alt="Screen Shot 2022-08-15 at 10 53 09 PM" src="https://user-images.githubusercontent.com/52147717/184794491-1f9412fd-4846-40bc-852e-68ab90eca28e.png">

I ran everything but cannot tell if it is being trained or not.
Ran the visdom at the start, using horse2zebra, then called the train. I am not sure what I am doing wrong here. Any help would be appreciative.

Thanks

**Comment by junyanz** [comment]
Are you running it on the CPU? If it is the case, the training will be extremely slow. You may want to set the `--print_freq 1` so that the program will print the loss more frequently.
